tools/proofers/review_work.php

```diff
@@ -410,16 +425,19 @@ function _echo_round_select($rounds, $selected)
     $diffLinkString = "";
     if ($sampleLimit > 0) {
         validate_projectID($projectid);
-        $query = sprintf("
+        $query = sprintf(
+            "
```

Comment:
`php-cs-fixer` won't change anything that isn't purely stylistic. It can move the `"` to the next line because whitespace before it is only stylistic, but it can't (and won't) change any of the content within the string because that is a functional change.

We could decide we want these a specific way and use this opportunity to format them the same way, but `php-cs-fixer` can't do that for us.

Reply:
I figured as much. The question is, is it worth worrying about in the short term (not this PR), or should we standardize those as we come to them? Or not bother?

Reply:
I have an update to standardize the SQL formatting across the codebase, but it will be much simpler to review that as a separate PR. With that in mind I'm going to merge this in and get an additional one open.